src/store/reducers.js

```diff
@@ -160,7 +175,8 @@ export const tableReducer = applyReducerHash(
 
 export const entitesDetailReducer = () => applyReducerHash(
     {
-        [INVENTORY_ACTION_TYPES.LOAD_ENTITY_FULFILLED]: entityLoaded
+        [INVENTORY_ACTION_TYPES.LOAD_ENTITY_FULFILLED]: entityLoaded,
+        [SET_ROS_TAB_VISBILITY]: resourceOptTabVisibility
```

Comment:
You can use `INVENTORY_ACTION_TYPES.LOAD_SYSTEM_PROFILE`
```suggestion
        [INVENTORY_ACTION_TYPES.LOAD_SYSTEM_PROFILE]: resourceOptTabVisibility
```